src/downloads.js

```diff
@@ -0,0 +1,16 @@
+
+export const downloads = {
+  acceptDanger: jest.fn((downloadId, cb) => cb()),
+  cancel: jest.fn((downloadId, cb) => cb()),
+  download: jest.fn((options, cb) => cb()),
+  erase: jest.fn((query, cb) => cb()),
+  getFileIcon: jest.fn((downloadId, cb) => cb()),
+  open: jest.fn(),
+  pause: jest.fn((downloadId, cb) => cb()),
+  removeFile: jest.fn((downloadId, cb) => cb()),
+  resume: jest.fn((downloadId, cb) => cb()),
+  search: jest.fn((query, cb) => cb()),
```

Comment:
Can you add the Promise support for all the callbacks here as well? See other examples of this: https://github.com/clarkbw/jest-webextension-mock/blob/02a4447578288b8536ad72729fe39c745f27b0e5/src/notifications.js#L1-L7 

It might be time to make that function more of a shared utility.